app/repositories/CampaignReferralRepository.scala

```diff
@@ -60,4 +60,12 @@ object CampaignReferralRepository {
     case "WINDOWS_NATIVE_APP" => "Windows"
     case other                => other
   }
+
+  private def formatPath(maybePath: Option[String]): String = {
+    maybePath match {
+      case Some(s) if s.startsWith("uk/") => s.stripPrefix("uk")
```

Comment:
I think app paths can be prefixed by any edition name, so probably need a case for `us`, `au` and whatever the name of the international edition is, possibly `international`.